cypress/support/commands.js

```diff
@@ -1,14 +1,20 @@
 import '@testing-library/cypress/add-commands';
 import { getIfExists } from '../helpers';
+
+const HOSTNAME = Cypress.env('BACKEND_HOST') || 'localhost';
+const GUILLOTINA_API_URL = `http://${HOSTNAME}:8081/db/web`;
+const PLONE_SITE_ID = Cypress.env('SITE_ID') || 'plone';
+const PLONE_API_URL = `http://${HOSTNAME}:55001/${PLONE_SITE_ID}`;
+
```

Comment:
Awesome! This will come in handy when configuring the Cypress GH Action in parallel mode, in a container infrastructure.

Since this change is related to Volto itself, can we have a CHANGELOG for this change in main Volto too? Thanks!

Reply:
Done!